### ISSUE TITLE
Do not hide status badge and release date on smaller resolutions

### ISSUE THREAD

**Issue by nlhkabu** [pypa/warehouse#6515]
On the project detail page, we currently hide the status badge and release date on mobile.  This is not accessible, as all information should be visible when users zoom in.
